documentation/en/README.md

```diff
@@ -0,0 +1,39 @@
+# GeoWebCache Documentation
+
+## Building
+
+An ant build script is required (note the python environment described below is required).
```

Comment:
An overview of the produced artifacts (e.g. target/html and *-docs.zip) would be nice here. Looking at the build.xml, they seem pretty consistent with other projects like GeoTools and GeoServer, but it'd still be a good addition, for completeness.